public/index.html

```diff
@@ -7,10 +7,25 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
+    <!-- Preconect to usual resources-->
+    <!-- Reference: https://www.keycdn.com/blog/resource-hints#preconnect -->
+    <link href="https://localhost:3000" rel="preconnect" crossorigin>
```

Comment:
is this for production use or only for local use?

Reply:
This is for `development` testing. Probably we would need to update that to `half-earth.vizzuality.com` before adding it to prod.

Reply:
Preferably this would be an env variable, so we never forget to update it.

Reply:
Yeah, that sounds like a good way of doing it.

Reply:
Thanks. I will add this `preconnect` resources dynamically (like the `preload` ones) as @sorodrigo says in the other comment.

Reply:
Sorry I mightve suggested something that's incorrect. The preconnect links should probably be server rendered, how ever if possible it would be nice to add them from a constants list of some sort, adding the API_URL as an env var.

In trase we're using a webpack plugin, perhaps you can make it work with CRA?
https://github.com/joaner/html-webpack-preconnect-plugin

If it's not possible to do it, just using a env var and writing it down in the index would be good enough ✌️

Reply:
Is there an immediate action regarding this in this PR? otherwise it would be good to go.